src/navigation/Stack.js

```diff
@@ -1,31 +1,42 @@
 import { createAppContainer, createStackNavigator } from 'react-navigation';
 
 // grab navigation
+import { BlurView } from 'expo';
+import { StyleSheet } from 'react-native';
+import React from 'react';
 import TabNavigation from './TabNavigation';
 
 // grab screens
 import ModalMusicPlayer from '../screens/ModalMusicPlayer';
+import ModalMoreOptions from '../screens/ModalMoreOptions';
 
 // grab modal routes (dynamic transitions)
 import ModalRoutes from './ModalRoutes';
 
 const StackNavigator = createStackNavigator(
   {
-    TabNavigation,
-
     // Modals
     // /////////////////////////////////////////////////////////////////////////
     ModalMusicPlayer: {
       screen: ModalMusicPlayer,
       navigationOptions: {
         gesturesEnabled: false
       }
-    }
+    },
+    ModalMoreOptions: {
+      screen: ModalMoreOptions,
+      navigationOptions: {
+        gesturesEnabled: false
+      }
+    },
+    TabNavigation
   },
   {
+    transparentCard: true,
     headerMode: 'none',
     initialRouteName: 'TabNavigation',
-    transitionConfig: ModalRoutes
+    // transitionConfig: ModalRoutes
+    mode: 'modal'
```

Comment:
add this `transitionConfig` back and remove `mode: modal` the alphabetize transparentCard in order.